docs/policy_templates_data/policy_templates.json

```diff
@@ -931,6 +931,34 @@
         ]
       }
     },
+    "AWSSecretsManagerGetSecretValueAccess": {
```

Comment:
The other policy templates end in Policy. How about `AWSSecretsManagerGetSecretValuePolicy`?